desktop/yast2-packager.desktop

```diff
@@ -8,6 +8,7 @@ Terminal=false
 Type=Application
 Categories=System;PackageManager;X-SuSE-ControlCenter-System;
 MimeType=application/x-rpm;application/x-redhat-package-manager;
+InitialPreference=10
```

Comment:
The [freedesktop documentation](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#kde-items) says that this keyword is KDE specific. Please add a comment with the documentation link and description that this is only for KDE and does not affect the other desktop environments.